src/collection-browser.ts

```diff
@@ -271,6 +278,14 @@ export class CollectionBrowser
     this.sortDirection = null;
   }
 
+  /**
+   * Manually requests to perform a search, which will only be executed if one of
+   * the query, the search type, or the sort has changed.
+   */
+  requestSearch() {
```

Comment:
Q: where does this get called?

Reply:
This is intended to be called by Offshoot code, so nowhere in collection-browser itself.

The use case is that someone has selected a different search type (say, switching from metadata to full text). Selecting that different option doesn't trigger a search on its own, but we still need a way to request that search later on when they hit Go, even if the search query itself hasn't changed.

Reply:
q: is there an interface that can represent this external api relationship by chance?

Reply:
I think there's a larger question here around more clearly specifying CollectionBrowser's external interface. At least from Offshoot's perspective, that interface also includes the `clearFilters` method and a number of other properties. We should definitely think about what we want this to look like going forward, but I'd rather limit this PR's scope and address these questions of architecture in the next one.